app/client/components/spinner.tsx

```diff
@@ -38,6 +40,7 @@ export const Spinner = (props: LoadingProps) => (
         height: scaledPx(40, props.scale),
         animation: "spin 2s linear infinite",
         margin: scaledPx(10, props.scale),
+        marginTop: props.alignCenter ? "center" : "",
```

Comment:
Should this be an alignment property rather than `margin-top` as it's being set to `center`?

Reply:
yeah, that's odd, it already has text-align center on it so removed this